### PR TITLE
Add WSL guide

### PR DESCRIPTION
**Pull request by minikN**
I'd appreciate feedback / suggestions / constructive criticism. I'll proof read it tomorrow but there are likely to be some typos.

Also I'd be great if someone could test it as well. As of now I've only tested it for myself.

### CODE REVIEW

File: content/guix/wsl.org [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+wsl -d guix /bin/busybox sh -c "/mnt/c/path/to/guix-install.sh /mnt/c/path/to/wsl.scm"
+#+end_src
+
+The path is relative to the root folder of the WSL distribution. If the two files are located at =C:\Users\<user>\Desktop\guix= the path would then be =/mnt/c/Users/<user>/Desktop/guix=.
+
+ *Note*: The install script and the manifest file don't have to be in the same folder. The script also sets the load path to the folder containing the manifest file, this means =wsl.scm= may inherit from other modules located in the same load path (like a =base-system.scm= for example).
+
+* Initialization
+
+After the installation is finished, it will most likely output a warning along the lines of
```
Comment (daviwil) [inline content/guix/wsl.org:192]: Looks like the expected warning text is missing here

File: content/guix/wsl.org [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+#+begin_src sh :tangle no
+wsl -d guix /bin/busybox sh -c "/mnt/c/path/to/guix-init.sh"
+#+end_src
+
+The command should not produce any output. This script /can/ be safely run every time one logs into the distribution (See [[#tidying-up][Tidying up]]), however, it /needs/ to be run after rebooting the host system or executing =wsl -t guix=. After it's done, run =wsl -d guix= to log into the distribution.
+
+A bash prompt should be waiting for input. Congratulations.
+
+No is a good time to set some passwords
```
Comment (daviwil) [inline content/guix/wsl.org:229]:
```suggestion
Now is a good time to set some passwords:
```

**Comment by daviwil** [comment]
Some notes from my installation experience:

- Need to link directly to the Busybox releases page, I went to the wrong place and couldn't find the rootfs.tgz file
- When I tried to create the new WSL 2 instance I received an error "The request is not supported", it turns out that I needed to turn on [Virtual Machine Platform](https://github.com/microsoft/WSL/issues/4840#issuecomment-584769225) in Windows features

More notes to come as I continue!

File: content/guix/wsl.org [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+The path is relative to the root folder of the WSL distribution. If the two files are located at =C:\Users\<user>\Desktop\guix= the path would then be =/mnt/c/Users/<user>/Desktop/guix=.
+
+ *Note*: The install script and the manifest file don't have to be in the same folder. The script also sets the load path to the folder containing the manifest file, this means =wsl.scm= may inherit from other modules located in the same load path (like a =base-system.scm= for example).
+
+* Initialization
+
+After the installation is finished, it will most likely output a warning along the lines of
+
+#+begin_comment
```
Comment (minikN) [inline content/guix/wsl.org:194]: Change this to code so github will display it

**Comment by minikN** [comment]
> Some notes from my installation experience:
> 
> * Need to link directly to the Busybox releases page, I went to the wrong place and couldn't find the rootfs.tgz file
> * When I tried to create the new WSL 2 instance I received an error "The request is not supported", it turns out that I needed to turn on [Virtual Machine Platform](https://github.com/microsoft/WSL/issues/4840#issuecomment-584769225) in Windows features
> 
> More notes to come as I continue!

Thanks @daviwil 

I fixed the first issue in a86432e4f.
I'm not sure about the second issue though. There is a [Prerequisites section](https://github.com/SystemCrafters/wiki-site/pull/28/files#diff-c1ba0364239ce2cc23c60679bf4e62e44cbc2b835c05441bb6d5f85fc6e4878dR20-R22) linking to the Microsoft docs for WSL2. I believe your issue is described there. Would you like to have the WSL2 setup directly in the guide instead of linking to the official docs?

**Pull request merged**